### DISCUSSION TITLE
QTL for Multitrait simulation

### DISCUSSION THREAD

**Discussion by marcopxt** [gaynorr/AlphaSimR#54] [General]
Hello, everyone!

I'm simulating a base population considering multitraits in AlphaSimR. The package give me the possibility to assign a different number of QTLs per chromosome, however I did not find the possibility of simulating different numbers of QTL per trait.

Is that a possibility?

Greatly appreciated,

**Comment by gaynorr** [comment]
There are currently two ways to do multiple traits. The first is to add multiple traits with a single call to addTrait_. This method lets you control the degree of correlation between QTL effects but it does require that all traits are of the same type and that they share the same QTL. The other method is to call addTrait_ multiple times. This method lets you mix trait types and differ the number of QTL for each trait. However, it doesn't let you control the degree of correlation between traits.
